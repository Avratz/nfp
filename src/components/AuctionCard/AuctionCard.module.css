.auctionCard {
  font-family: var(--font-poppins);
  background-color: var(--neutral-2);
  border-radius: 1.25rem;
  padding: 0.75rem;
}

.image {
  width: 100%;
  aspect-ratio: 11 / 12;
  position: relative;
  border-radius: 1rem;
  overflow: hidden;
}

.auctionCardData {
  margin: 1.25rem 0;
  display: flex;
  flex-direction: column;
  align-items: center;
  text-align: center;
  gap: 0.75rem;
}

.headerTitle, .stock, .highestBidBox{
  width: 100%;
  display: flex;
  justify-content: space-between;
  align-items: center;
}

.title {
  font-weight: var(--font-medium);
}

.instantPrice{
  font-weight: var(--font-bold);
  font-size: var(--text-xs);
  line-height: var(--line-xs);
  border: 2px solid var(--green);
  color: var(--green);
  border-radius: 0.25rem;
  padding: 0.375rem;
  padding-bottom: 0.25rem;
}

.stockAvatars {
  display: flex;
}

.stockAvatars picture:not(:first-child) {
  margin-left: -0.5rem;
}

.stockValue{
  font-weight: var(--font-medium);
  font-size: var(--text-sm);
  color: var(--neutral-6);
}

.divider{
  width:100%;
  height: 1px;
  background-color: var(--neutral-3);
}

.highestBidBox {
  font-weight: var(--font-regular);
  color: var(--neutral-4);
  font-size: var(--text-xs);
}

.highestBid{
  display: flex;
  align-items: center;
  gap: 0.25rem;
}

.highestBidValue {
  font-weight: var(--font-medium);
  color: var(--neutral-6);
}