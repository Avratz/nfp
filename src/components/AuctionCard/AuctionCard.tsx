import { Avatar } from '@/ui/Avatar/Avatar'
import { Divider } from '@/ui/Divider/Divider'
import { Heading } from '@/ui/Heading/Heading'
import { Icon } from '@/ui/Icon/Icon'
import Image from 'next/image'

import styles from './AuctionCard.module.css'

type AuctionCardProps = {
  image: string
  author: string
  instantPrice: string
  stock: number
  authorAvatar: string
  highestBid: string
}

export function AuctionCard({
  image,
  author,
  instantPrice,
  stock,
  authorAvatar,
  highestBid,
}: AuctionCardProps) {
  return (
    <article className={styles.auctionCard}>
      <picture className={styles.image}>
        <Image
          src={image}
          alt=""
          fill
          sizes="(max-width: 768px) 100vw,
              33vw"
          style={{
            objectFit: 'cover',
          }}
        />
      </picture>
      <div className={styles.auctionCardData}>
        <header className={styles.headerTitle}>
          <Heading as="h3" className={styles.title}>
            {author} NFT
          </Heading>
          <span className={styles.instantPrice}>{instantPrice}</span>
        </header>
        <div className={styles.stock}>
          <span className={styles.stockAvatars}>
            {Array.from({ length: stock }).map((_, index) => (
              <Avatar src={authorAvatar} size="small" key={index} />
            ))}
          </span>
          <span className={styles.stockValue}>{stock} in stock</span>
        </div>
        <Divider />
        <div className={styles.highestBidBox}>
          <span className={styles.highestBid}>
            <Icon name="candlesticks" />
            <span>Highest bid </span>
            <span className={styles.highestBidValue}>{highestBid}</span>
          </span>
          <span>New bid ðŸ”¥</span>
        </div>
      </div>
    </article>
  )
}
