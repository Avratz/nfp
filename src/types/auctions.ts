export type Auction = {
  id: number
  instantPrice: string
  highestBid: string
  author: string
  authorAvatar: string
  type: string
  stock: number
  likes: number
  createdAt: Date
  endsAt: Date
  media: {
    id: number
    image: string
    image2x: string
  }
  attributes: {
    id: number
    color: string
    type: string
  }
  bidUsers: [
    {
      id: number
      name: string
      avatar: string
    },
  ]
}
