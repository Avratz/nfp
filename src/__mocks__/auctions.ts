import { Auction } from '@/types/auctions'

export const allAuctions: Auction[] = [
  {
    id: 10,
    instantPrice: '4.45 ETH',
    highestBid: '0.010 ETH',
    author: 'Fernando Medina',
    authorAvatar:
      'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/07_xrihyb.png',
    type: 'Art',
    stock: 1,
    likes: 2,
    createdAt: '2022-06-05T11:42:33.000Z',
    endsAt: '2022-08-06T11:42:33.000Z',
    media: {
      id: 4,
      image:
        'https://res.cloudinary.com/dvmll0ruo/image/upload/c_scale,w_1000/v1657121559/NFPAISANOS/img_125_mrxfbh.png',
      image2x:
        'https://res.cloudinary.com/dvmll0ruo/image/upload/c_scale,w_1500/v1657121559/NFPAISANOS/img_125_mrxfbh.png',
    },
    attributes: {
      id: 4,
      color: 'black',
      type: 'epic',
    },
    bidUsers: [
      {
        id: 6,
        name: 'Sofia Coolen',
        avatar:
          'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/05_rdi4su.png',
      },
      {
        id: 7,
        name: 'Martin Acosta',
        avatar:
          'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/04_uunxgl.png',
      },
    ],
  },
  {
    id: 11,
    instantPrice: '2.45 ETH',
    highestBid: '0.100 ETH',
    author: 'Martin Acosta',
    authorAvatar:
      'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/06_mjddoi.png',
    type: 'Art',
    stock: 3,
    likes: 10,
    createdAt: '2022-06-06T11:42:33.000Z',
    endsAt: '2022-08-06T11:42:33.000Z',
    media: {
      id: 5,
      image:
        'https://res.cloudinary.com/dvmll0ruo/image/upload/c_scale,w_1000/v1657121554/NFPAISANOS/img_131_d7wmyk.png',
      image2x:
        'https://res.cloudinary.com/dvmll0ruo/image/upload/c_scale,w_1500/v1657121559/NFPAISANOS/img_125_mrxfbh.png',
    },
    attributes: {
      id: 5,
      color: 'pink',
      type: 'rare',
    },
    bidUsers: [
      {
        id: 8,
        name: 'Nicolas Sieiro',
        avatar:
          'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/02_nqwu5e.png',
      },
      {
        id: 9,
        name: 'Carla Luna',
        avatar:
          'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/03_ifygi0.png',
      },
    ],
  },
  {
    id: 12,
    instantPrice: '2.15 ETH',
    highestBid: '0.500 ETH',
    author: 'Santiago Lopez',
    authorAvatar:
      'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/04_uunxgl.png',
    type: 'Photography',
    stock: 5,
    likes: 10,
    createdAt: '2022-06-07T11:42:33.000Z',
    endsAt: '2022-08-06T11:42:33.000Z',
    media: {
      id: 6,
      image:
        'https://res.cloudinary.com/dvmll0ruo/image/upload/c_scale,w_1000/v1657121551/NFPAISANOS/img_126_ae6zew.png',
      image2x:
        'https://res.cloudinary.com/dvmll0ruo/image/upload/c_scale,w_1000/v1657121551/NFPAISANOS/img_126_ae6zew.png',
    },
    attributes: {
      id: 6,
      color: 'orange',
      type: 'uncommon',
    },
    bidUsers: [
      {
        id: 10,
        name: 'Martin Acosta',
        avatar:
          'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/07_xrihyb.png',
      },
      {
        id: 11,
        name: 'Antonella Richieri',
        avatar:
          'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/08_fsbhry.png',
      },
    ],
  },
  {
    id: 13,
    instantPrice: '10.05 ETH',
    highestBid: '1.500 ETH',
    author: 'Santiago Echazu',
    authorAvatar:
      'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/01_qgg62f.png',
    type: 'Art',
    stock: 1,
    likes: 30,
    createdAt: '2022-06-10T11:42:33.000Z',
    endsAt: '2022-08-06T11:42:33.000Z',
    media: {
      id: 7,
      image:
        'https://res.cloudinary.com/dvmll0ruo/image/upload/c_scale,w_1000/v1657121542/NFPAISANOS/img_101_y2iczn.png',
      image2x:
        'https://res.cloudinary.com/dvmll0ruo/image/upload/c_scale,w_1500/v1657121542/NFPAISANOS/img_101_y2iczn.png',
    },
    attributes: {
      id: 7,
      color: 'orange',
      type: 'legendary',
    },
    bidUsers: [
      {
        id: 12,
        name: 'Martin Acosta',
        avatar:
          'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/07_xrihyb.png',
      },
      {
        id: 13,
        name: 'Antonella Richieri',
        avatar:
          'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/08_fsbhry.png',
      },
      {
        id: 14,
        name: 'Antonella Richieri',
        avatar:
          'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/08_fsbhry.png',
      },
      {
        id: 15,
        name: 'Martin Acosta',
        avatar:
          'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/07_xrihyb.png',
      },
    ],
  },
  {
    id: 14,
    instantPrice: '1.05 ETH',
    highestBid: '0.40 ETH',
    author: 'Alvaro Echazu',
    authorAvatar:
      'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/02_nqwu5e.png',
    type: 'Art',
    stock: 1,
    likes: 400,
    createdAt: '2022-06-15T11:42:33.000Z',
    endsAt: '2022-08-06T11:42:33.000Z',
    media: {
      id: 8,
      image:
        'https://res.cloudinary.com/dvmll0ruo/image/upload/c_scale,w_1000/v1657121536/NFPAISANOS/img_102_a9rkb7.png',
      image2x:
        'https://res.cloudinary.com/dvmll0ruo/image/upload/c_scale,w_1000/v1657121536/NFPAISANOS/img_102_a9rkb7.png',
    },
    attributes: {
      id: 8,
      color: 'black',
      type: 'legendary',
    },
    bidUsers: [
      {
        id: 16,
        name: 'Akira Kaneto',
        avatar:
          'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/04_uunxgl.png',
      },
      {
        id: 17,
        name: 'Facundo Aguinaga',
        avatar:
          'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/04_uunxgl.png',
      },
    ],
  },
  {
    id: 15,
    instantPrice: '1.30 ETH',
    highestBid: '0.022 ETH',
    author: 'Sofia Coolen',
    authorAvatar:
      'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/03_ifygi0.png',
    type: 'Photography',
    stock: 2,
    likes: 14,
    createdAt: '2022-06-20T11:42:33.000Z',
    endsAt: '2022-08-06T11:42:33.000Z',
    media: {
      id: 9,
      image:
        'https://res.cloudinary.com/dvmll0ruo/image/upload/c_scale,w_1000/v1657121534/NFPAISANOS/img_115_lolex4.png',
      image2x:
        'https://res.cloudinary.com/dvmll0ruo/image/upload/c_scale,w_1500/v1657121534/NFPAISANOS/img_115_lolex4.png',
    },
    attributes: {
      id: 9,
      color: 'blue',
      type: 'uncommon',
    },
    bidUsers: [
      {
        id: 18,
        name: 'Julian Krupka',
        avatar:
          'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/01_qgg62f.png',
      },
      {
        id: 19,
        name: 'Julian Alvarez',
        avatar:
          'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/04_uunxgl.png',
      },
    ],
  },
  {
    id: 16,
    instantPrice: '6.52 ETH',
    highestBid: '0.031 ETH',
    author: 'Flora Quiroga',
    authorAvatar:
      'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/08_fsbhry.png',
    type: 'Photography',
    stock: 1,
    likes: 201,
    createdAt: '2022-06-30T11:42:33.000Z',
    endsAt: '2022-08-06T11:42:33.000Z',
    media: {
      id: 10,
      image:
        'https://res.cloudinary.com/dvmll0ruo/image/upload/c_scale,w_1000/v1657121534/NFPAISANOS/img_119_o3ouhw.png',
      image2x:
        'https://res.cloudinary.com/dvmll0ruo/image/upload/c_scale,w_1000/v1657121534/NFPAISANOS/img_119_o3ouhw.png',
    },
    attributes: {
      id: 10,
      color: 'pink',
      type: 'legendary',
    },
    bidUsers: [
      {
        id: 20,
        name: 'Manuel Torres',
        avatar:
          'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/04_uunxgl.png',
      },
      {
        id: 21,
        name: 'Franco Kaus',
        avatar:
          'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/02_nqwu5e.png',
      },
    ],
  },
  {
    id: 17,
    instantPrice: '0.002 ETH',
    highestBid: '0.00005 ETH',
    author: 'Mar√≠a Del Mar Gestoso',
    authorAvatar:
      'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/03_ifygi0.png',
    type: 'Art',
    stock: 1,
    likes: 0,
    createdAt: '2022-06-01T11:10:33.000Z',
    endsAt: '2022-09-06T11:42:33.000Z',
    media: {
      id: 11,
      image:
        'https://res.cloudinary.com/dvmll0ruo/image/upload/c_scale,w_1000/v1657121534/NFPAISANOS/img_133_slwzia.png',
      image2x:
        'https://res.cloudinary.com/dvmll0ruo/image/upload/c_scale,w_1500/v1657121534/NFPAISANOS/img_133_slwzia.png',
    },
    attributes: {
      id: 11,
      color: 'red',
      type: 'uncommon',
    },
    bidUsers: [],
  },
  {
    id: 18,
    instantPrice: '0.028 ETH',
    highestBid: '0.00052 ETH',
    author: 'Julian Cerino',
    authorAvatar:
      'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/01_qgg62f.png',
    type: 'Art',
    stock: 10,
    likes: 5,
    createdAt: '2022-06-14T11:10:33.000Z',
    endsAt: '2022-08-06T11:42:33.000Z',
    media: {
      id: 12,
      image:
        'https://res.cloudinary.com/dvmll0ruo/image/upload/c_scale,w_1000/v1657121534/NFPAISANOS/img_114_d5usms.png',
      image2x:
        'https://res.cloudinary.com/dvmll0ruo/image/upload/c_scale,w_1500/v1657121534/NFPAISANOS/img_114_d5usms.png',
    },
    attributes: {
      id: 12,
      color: 'black',
      type: 'rare',
    },
    bidUsers: [
      {
        id: 22,
        name: 'Leo Suarez',
        avatar:
          'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/07_xrihyb.png',
      },
    ],
  },
  {
    id: 19,
    instantPrice: '1.00 ETH',
    highestBid: '0.012 ETH',
    author: 'Martin Russo',
    authorAvatar:
      'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/06_mjddoi.png',
    type: 'Art',
    stock: 1,
    likes: 30,
    createdAt: '2022-06-10T11:10:33.000Z',
    endsAt: '2022-07-06T11:42:33.000Z',
    media: {
      id: 13,
      image:
        'https://res.cloudinary.com/dvmll0ruo/image/upload/c_scale,w_1000/v1657121532/NFPAISANOS/img_122_s2bk3s.png',
      image2x:
        'https://res.cloudinary.com/dvmll0ruo/image/upload/c_scale,w_1500/v1657121532/NFPAISANOS/img_122_s2bk3s.png',
    },
    attributes: {
      id: 13,
      color: 'black',
      type: 'legendary',
    },
    bidUsers: [
      {
        id: 23,
        name: 'Juan Lazo',
        avatar:
          'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/07_xrihyb.png',
      },
      {
        id: 24,
        name: 'Exequiel Germano',
        avatar:
          'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/02_nqwu5e.png',
      },
    ],
  },
  {
    id: 20,
    instantPrice: '3.00 ETH',
    highestBid: '0.21 ETH',
    author: 'Belen Lago',
    authorAvatar:
      'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/05_rdi4su.png',
    type: 'Photography',
    stock: 5,
    likes: 15,
    createdAt: '2022-06-03T11:10:33.000Z',
    endsAt: '2022-07-06T11:42:33.000Z',
    media: {
      id: 14,
      image:
        'https://res.cloudinary.com/dvmll0ruo/image/upload/c_scale,w_1000/v1657121534/NFPAISANOS/img_104_zklzrv.png',
      image2x:
        'https://res.cloudinary.com/dvmll0ruo/image/upload/c_scale,w_1500/v1657121534/NFPAISANOS/img_104_zklzrv.png',
    },
    attributes: {
      id: 14,
      color: 'orange',
      type: 'rare',
    },
    bidUsers: [
      {
        id: 25,
        name: 'Nacho Margulies',
        avatar:
          'https://res.cloudinary.com/dvmll0ruo/image/upload/v1657153929/NFPAISANOS/07_xrihyb.png',
      },
    ],
  },
]
