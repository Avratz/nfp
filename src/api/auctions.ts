import { getCachedOneMinute } from '@/lib/fetcher'
import { Auction } from '@/types/auctions'

export function getPopularAuctions(): Promise<Auction[]> {
  return getCachedOneMinute('/popular', process.env.API_KEY as string)
}

export function getAllAuctions(): Promise<Auction[]> {
  return getCachedOneMinute('/aunctions', process.env.API_KEY as string)
}
