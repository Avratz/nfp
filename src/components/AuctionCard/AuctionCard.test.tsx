import { render } from '@testing-library/react'
import { AuctionCard } from './AuctionCard'

import { allAuctions } from '@/__mocks__/auctions'

jest.mock('@/ui/Icon/Icon', () => {
  function Icon() {
    return <div></div>
  }

  Icon.displayName = 'Icon'
  return {
    __esModule: true,
    Icon,
  }
})

const auction = allAuctions[0]
describe('AuctionCard', () => {
  it('renders AuctionCard', () => {
    let { container } = render(
      <AuctionCard
        image={auction.media.image}
        author={auction.author}
        instantPrice={auction.instantPrice}
        stock={auction.stock}
        highestBid={auction.highestBid}
        bidUsers={auction.bidUsers}
        attributeType={auction.attributes.type}
      />,
    )
    expect(container).toMatchSnapshot()
  })
})
