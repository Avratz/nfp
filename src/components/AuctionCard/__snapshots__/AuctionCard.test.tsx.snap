// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuctionCard renders AuctionCard 1`] = `
<div>
  <article
    class="auctionCard"
    style="opacity: 0;"
  >
    <picture
      class="image"
    >
      <div
        class="layerOver"
      >
        <div
          class="auctionInfoHover"
        >
          <div
            class="badge solid"
          >
            epic
          </div>
          <button
            class="button icon base heartButton"
            title="Like"
          >
            <div />
          </button>
        </div>
        <button
          class="button accent base placeABidBtn"
        >
          <span>
            Place a bid
          </span>
           
          <div />
        </button>
      </div>
      <img
        alt=""
        data-nimg="fill"
        decoding="async"
        loading="lazy"
        sizes="(max-width: 768px) 40vw, 15vw"
        src="/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fc_scale%2Cw_1000%2Fv1657121559%2FNFPAISANOS%2Fimg_125_mrxfbh.png&w=3840&q=65"
        srcset="/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fc_scale%2Cw_1000%2Fv1657121559%2FNFPAISANOS%2Fimg_125_mrxfbh.png&w=96&q=65 96w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fc_scale%2Cw_1000%2Fv1657121559%2FNFPAISANOS%2Fimg_125_mrxfbh.png&w=128&q=65 128w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fc_scale%2Cw_1000%2Fv1657121559%2FNFPAISANOS%2Fimg_125_mrxfbh.png&w=256&q=65 256w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fc_scale%2Cw_1000%2Fv1657121559%2FNFPAISANOS%2Fimg_125_mrxfbh.png&w=384&q=65 384w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fc_scale%2Cw_1000%2Fv1657121559%2FNFPAISANOS%2Fimg_125_mrxfbh.png&w=640&q=65 640w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fc_scale%2Cw_1000%2Fv1657121559%2FNFPAISANOS%2Fimg_125_mrxfbh.png&w=750&q=65 750w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fc_scale%2Cw_1000%2Fv1657121559%2FNFPAISANOS%2Fimg_125_mrxfbh.png&w=828&q=65 828w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fc_scale%2Cw_1000%2Fv1657121559%2FNFPAISANOS%2Fimg_125_mrxfbh.png&w=1080&q=65 1080w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fc_scale%2Cw_1000%2Fv1657121559%2FNFPAISANOS%2Fimg_125_mrxfbh.png&w=1200&q=65 1200w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fc_scale%2Cw_1000%2Fv1657121559%2FNFPAISANOS%2Fimg_125_mrxfbh.png&w=1920&q=65 1920w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fc_scale%2Cw_1000%2Fv1657121559%2FNFPAISANOS%2Fimg_125_mrxfbh.png&w=2048&q=65 2048w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fc_scale%2Cw_1000%2Fv1657121559%2FNFPAISANOS%2Fimg_125_mrxfbh.png&w=3840&q=65 3840w"
        style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: cover; color: transparent;"
      />
    </picture>
    <div
      class="auctionCardData"
    >
      <header
        class="headerTitle"
      >
        <h3
          class="heading base title"
        >
          Fernando Medina
           NFT
        </h3>
        <div
          class="badge outline"
        >
          4.45 ETH
        </div>
      </header>
      <div
        class="stock"
      >
        <span
          class="stockAvatars"
        >
          <picture
            class="avatar small"
          >
            <img
              alt=""
              data-nimg="1"
              decoding="async"
              height="20"
              loading="lazy"
              sizes="100vw"
              src="/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fv1657153929%2FNFPAISANOS%2F05_rdi4su.png&w=3840&q=75"
              srcset="/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fv1657153929%2FNFPAISANOS%2F05_rdi4su.png&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fv1657153929%2FNFPAISANOS%2F05_rdi4su.png&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fv1657153929%2FNFPAISANOS%2F05_rdi4su.png&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fv1657153929%2FNFPAISANOS%2F05_rdi4su.png&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fv1657153929%2FNFPAISANOS%2F05_rdi4su.png&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fv1657153929%2FNFPAISANOS%2F05_rdi4su.png&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fv1657153929%2FNFPAISANOS%2F05_rdi4su.png&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fv1657153929%2FNFPAISANOS%2F05_rdi4su.png&w=3840&q=75 3840w"
              style="color: transparent;"
              width="20"
            />
          </picture>
          <picture
            class="avatar small"
          >
            <img
              alt=""
              data-nimg="1"
              decoding="async"
              height="20"
              loading="lazy"
              sizes="100vw"
              src="/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fv1657153929%2FNFPAISANOS%2F04_uunxgl.png&w=3840&q=75"
              srcset="/_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fv1657153929%2FNFPAISANOS%2F04_uunxgl.png&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fv1657153929%2FNFPAISANOS%2F04_uunxgl.png&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fv1657153929%2FNFPAISANOS%2F04_uunxgl.png&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fv1657153929%2FNFPAISANOS%2F04_uunxgl.png&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fv1657153929%2FNFPAISANOS%2F04_uunxgl.png&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fv1657153929%2FNFPAISANOS%2F04_uunxgl.png&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fv1657153929%2FNFPAISANOS%2F04_uunxgl.png&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fres.cloudinary.com%2Fdvmll0ruo%2Fimage%2Fupload%2Fv1657153929%2FNFPAISANOS%2F04_uunxgl.png&w=3840&q=75 3840w"
              style="color: transparent;"
              width="20"
            />
          </picture>
        </span>
        <span
          class="stockValue"
        >
          1
           in stock
        </span>
      </div>
      <hr
        class="divider horizontal"
      />
      <div
        class="highestBidBox"
      >
        <span
          class="highestBid"
        >
          <div />
          <span>
            Highest bid 
          </span>
          <span
            class="highestBidValue"
          >
            0.010 ETH
          </span>
        </span>
        <span>
          New bid ðŸ”¥
        </span>
      </div>
    </div>
  </article>
</div>
`;
